src/renderers/art/ReactARTFiberEntry.js

```diff
@@ -509,9 +509,7 @@ const ARTRenderer = ReactFiberReconciler({
     return emptyObject;
   },
 
-  scheduleAnimationCallback: ReactDOMFrameScheduling.rAF,
-
-  scheduleDeferredCallback: ReactDOMFrameScheduling.rIC,
+  scheduleCallback: ReactDOMFrameScheduling.rIC,
```

Comment:
Remember that this is a __public__ API for renderers to use. It should first and foremost describe what kind of function _they're_ supposed to provide here.

Async isn't quite enough since that could be a micro-task tick (like the Promise queue).